ext/curl/interface.c

```diff
@@ -2466,7 +2465,7 @@ PHP_FUNCTION(curl_exec)
 
 	if (!Z_ISUNDEF(ch->handlers.std_err)) {
 		php_stream  *stream;
-		stream = (php_stream*)zend_fetch_resource2_ex(&ch->handlers.std_err, NULL, php_file_le_stream(), php_file_le_pstream());
+		php_stream_from_zval_no_verify(stream, &ch->handlers.std_err);
```

Comment:
nit: I think you would need to change other instances because now you have named resource, i.e. stream, wdyt ?

Reply:
Ah right, I somehow missed them.
I'll split this into a different commit.